test/unit/modules/peers.js

```diff
@@ -927,7 +926,7 @@ describe('peers', () => {
 		});
 
 		afterEach(() => {
-			return revertPrivateStubs();
+			return revertPrivateStubs;
```

Comment:
What does this part do now? Is it even needed?

Reply:
I think the variable it self is unused, for some reason @LucasIsasmendi changed it, but i am removing this variable and afterEach hook as the variable is not being used anywhere and all the tests are working.